src/postfixCompletionProvider.ts

```diff
@@ -43,6 +43,7 @@ export class PostfixCompletionProvider implements vsc.CompletionItemProvider {
     }
 
     const indentSize = this.getIndentSize(document, currentNode)
+    const leadingWhitespace = this.getLeadingWhitespace(document, currentNode)

```

Comment:
How about merging those two into `getIndentInfo`? They are very similar inside.

Reply:
To be honest I didn't even notice this somehow. Thank you for suggestion!